Tribler/Test/Core/Modules/RestApi/test_display_endpoint.py

```diff
@@ -59,12 +75,7 @@ def test_get_no_edges(self):
         exp_message = {"focus_node": "30", "neighbor_level": 1, "nodes": [{"public_key": "xyz", "total_up": 0,
                                                                            "total_down": 0, "page_rank": 0.5}],
                        "edges": []}
-        mocked_session = MockObject()
-        display_endpoint = DisplayEndpoint(mocked_session)
-        display_endpoint.get_multi_chain_community = lambda: self.mc_community
-        request = MockObject()
-        request.setHeader = lambda header, flags: None
-        request.args = {"focus_node": ['30'], "neighbor_level": ['1']}
+        display_endpoint, request = self.set_up_endpoint_request(30, 1)
         self.assertEqual(dumps(exp_message), display_endpoint.render_GET(request))
```

Comment:
As you said last week, you could just `import json` and change this into `json.dumps()` for readability